alchemist-implementationbase/src/main/kotlin/it/unibo/alchemist/util/Environments.kt

```diff
@@ -130,19 +115,15 @@ object Environments {
             neighborDistanceMetric { n1, n2 ->
                 getDistanceBetweenNodes(n1, n2)
             },
-    ): Map<UndirectedEdge<T>, Double> {
+    ): MutableDoubleSymmetricMatrix {
```

Comment:
```suggestion
    ): DoubleSymmetricMatrix {
```

Don't propagate mutability unnecessarily